<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="4" ignored="0" total="4" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Automation Practise Suite" duration-ms="14056" started-at="2020-06-24T09:16:37Z" finished-at="2020-06-24T09:16:51Z">
    <groups>
    </groups>
    <test name="Chrome Local Test" duration-ms="8820" started-at="2020-06-24T09:16:37Z" finished-at="2020-06-24T09:16:45Z">
      <class name="com.Automation.TestCases.LaunchPageTest">
        <test-method status="PASS" signature="setUpTestCase(java.lang.String, java.lang.String)[pri:0, instance:com.Automation.TestCases.LaunchPageTest@1bc0d349]" name="setUpTestCase" is-config="true" duration-ms="5103" started-at="2020-06-24T09:16:37Z" finished-at="2020-06-24T09:16:42Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[local]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpTestCase -->
        <test-method status="PASS" signature="setUpTestMethod()[pri:0, instance:com.Automation.TestCases.LaunchPageTest@1bc0d349]" name="setUpTestMethod" is-config="true" duration-ms="14" started-at="2020-06-24T09:16:42Z" finished-at="2020-06-24T09:16:42Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpTestMethod -->
        <test-method status="FAIL" signature="tc_VerifyClickOnSignIn()[pri:1, instance:com.Automation.TestCases.LaunchPageTest@1bc0d349]" name="tc_VerifyClickOnSignIn" duration-ms="1569" started-at="2020-06-24T09:16:42Z" description="Verify Authentication Page is Opened on click on SignIn link" finished-at="2020-06-24T09:16:43Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected:<Login - My Store> but was:<Login - My Stor>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected:<Login - My Store> but was:<Login - My Stor>
at org.testng.AssertJUnit.fail(AssertJUnit.java:59)
at org.testng.AssertJUnit.failNotEquals(AssertJUnit.java:364)
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:80)
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:88)
at com.Automation.TestCases.LaunchPageTest.tc_VerifyClickOnSignIn(LaunchPageTest.java:35)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc_VerifyClickOnSignIn -->
        <test-method status="PASS" signature="tearDownTestCase()[pri:0, instance:com.Automation.TestCases.LaunchPageTest@1bc0d349]" name="tearDownTestCase" is-config="true" duration-ms="100" started-at="2020-06-24T09:16:45Z" finished-at="2020-06-24T09:16:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownTestCase -->
      </class> <!-- com.Automation.TestCases.LaunchPageTest -->
    </test> <!-- Chrome Local Test -->
    <test name="Firefox Local Test" duration-ms="9601" started-at="2020-06-24T09:16:37Z" finished-at="2020-06-24T09:16:46Z">
      <class name="com.Automation.TestCases.LaunchPageTest">
        <test-method status="PASS" signature="setUpTestCase(java.lang.String, java.lang.String)[pri:0, instance:com.Automation.TestCases.LaunchPageTest@5b3a7ef5]" name="setUpTestCase" is-config="true" duration-ms="6692" started-at="2020-06-24T09:16:37Z" finished-at="2020-06-24T09:16:43Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[local]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[firefox]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpTestCase -->
        <test-method status="FAIL" signature="tc_VerifyClickOnSignIn()[pri:1, instance:com.Automation.TestCases.LaunchPageTest@5b3a7ef5]" name="tc_VerifyClickOnSignIn" duration-ms="1406" started-at="2020-06-24T09:16:43Z" description="Verify Authentication Page is Opened on click on SignIn link" finished-at="2020-06-24T09:16:45Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected:<Login - My Store> but was:<Login - My Stor>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected:<Login - My Store> but was:<Login - My Stor>
at org.testng.AssertJUnit.fail(AssertJUnit.java:59)
at org.testng.AssertJUnit.failNotEquals(AssertJUnit.java:364)
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:80)
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:88)
at com.Automation.TestCases.LaunchPageTest.tc_VerifyClickOnSignIn(LaunchPageTest.java:35)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc_VerifyClickOnSignIn -->
        <test-method status="PASS" signature="setUpTestMethod()[pri:0, instance:com.Automation.TestCases.LaunchPageTest@5b3a7ef5]" name="setUpTestMethod" is-config="true" duration-ms="0" started-at="2020-06-24T09:16:43Z" finished-at="2020-06-24T09:16:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpTestMethod -->
        <test-method status="PASS" signature="tearDownTestCase()[pri:0, instance:com.Automation.TestCases.LaunchPageTest@5b3a7ef5]" name="tearDownTestCase" is-config="true" duration-ms="555" started-at="2020-06-24T09:16:46Z" finished-at="2020-06-24T09:16:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownTestCase -->
      </class> <!-- com.Automation.TestCases.LaunchPageTest -->
    </test> <!-- Firefox Local Test -->
    <test name="Firefox Remote Test" duration-ms="13444" started-at="2020-06-24T09:16:37Z" finished-at="2020-06-24T09:16:50Z">
      <class name="com.Automation.TestCases.LaunchPageTest">
        <test-method status="PASS" signature="setUpTestCase(java.lang.String, java.lang.String)[pri:0, instance:com.Automation.TestCases.LaunchPageTest@fab35b1]" name="setUpTestCase" is-config="true" duration-ms="8765" started-at="2020-06-24T09:16:37Z" finished-at="2020-06-24T09:16:45Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[remote]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[firefox]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpTestCase -->
        <test-method status="PASS" signature="setUpTestMethod()[pri:0, instance:com.Automation.TestCases.LaunchPageTest@fab35b1]" name="setUpTestMethod" is-config="true" duration-ms="1" started-at="2020-06-24T09:16:45Z" finished-at="2020-06-24T09:16:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpTestMethod -->
        <test-method status="FAIL" signature="tc_VerifyClickOnSignIn()[pri:1, instance:com.Automation.TestCases.LaunchPageTest@fab35b1]" name="tc_VerifyClickOnSignIn" duration-ms="1817" started-at="2020-06-24T09:16:45Z" description="Verify Authentication Page is Opened on click on SignIn link" finished-at="2020-06-24T09:16:47Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected:<Login - My Store> but was:<Login - My Stor>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected:<Login - My Store> but was:<Login - My Stor>
at org.testng.AssertJUnit.fail(AssertJUnit.java:59)
at org.testng.AssertJUnit.failNotEquals(AssertJUnit.java:364)
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:80)
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:88)
at com.Automation.TestCases.LaunchPageTest.tc_VerifyClickOnSignIn(LaunchPageTest.java:35)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc_VerifyClickOnSignIn -->
        <test-method status="PASS" signature="tearDownTestCase()[pri:0, instance:com.Automation.TestCases.LaunchPageTest@fab35b1]" name="tearDownTestCase" is-config="true" duration-ms="1076" started-at="2020-06-24T09:16:49Z" finished-at="2020-06-24T09:16:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownTestCase -->
      </class> <!-- com.Automation.TestCases.LaunchPageTest -->
    </test> <!-- Firefox Remote Test -->
    <test name="Chrome Remote Test" duration-ms="14056" started-at="2020-06-24T09:16:37Z" finished-at="2020-06-24T09:16:51Z">
      <class name="com.Automation.TestCases.LaunchPageTest">
        <test-method status="PASS" signature="setUpTestCase(java.lang.String, java.lang.String)[pri:0, instance:com.Automation.TestCases.LaunchPageTest@34d52ecd]" name="setUpTestCase" is-config="true" duration-ms="9340" started-at="2020-06-24T09:16:37Z" finished-at="2020-06-24T09:16:46Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[remote]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpTestCase -->
        <test-method status="PASS" signature="setUpTestMethod()[pri:0, instance:com.Automation.TestCases.LaunchPageTest@34d52ecd]" name="setUpTestMethod" is-config="true" duration-ms="0" started-at="2020-06-24T09:16:46Z" finished-at="2020-06-24T09:16:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpTestMethod -->
        <test-method status="FAIL" signature="tc_VerifyClickOnSignIn()[pri:1, instance:com.Automation.TestCases.LaunchPageTest@34d52ecd]" name="tc_VerifyClickOnSignIn" duration-ms="2126" started-at="2020-06-24T09:16:46Z" description="Verify Authentication Page is Opened on click on SignIn link" finished-at="2020-06-24T09:16:48Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected:<Login - My Store> but was:<Login - My Stor>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected:<Login - My Store> but was:<Login - My Stor>
at org.testng.AssertJUnit.fail(AssertJUnit.java:59)
at org.testng.AssertJUnit.failNotEquals(AssertJUnit.java:364)
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:80)
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:88)
at com.Automation.TestCases.LaunchPageTest.tc_VerifyClickOnSignIn(LaunchPageTest.java:35)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc_VerifyClickOnSignIn -->
        <test-method status="PASS" signature="tearDownTestCase()[pri:0, instance:com.Automation.TestCases.LaunchPageTest@34d52ecd]" name="tearDownTestCase" is-config="true" duration-ms="339" started-at="2020-06-24T09:16:50Z" finished-at="2020-06-24T09:16:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownTestCase -->
      </class> <!-- com.Automation.TestCases.LaunchPageTest -->
    </test> <!-- Chrome Remote Test -->
  </suite> <!-- Automation Practise Suite -->
</testng-results>
